docs/open-source-tizen/tools/gbs/gbs-devel.md

```diff
@@ -209,26 +192,19 @@ This command facilitates package maintainers to better maintain packaging branch
   upstream
   ```
 
-  The convert action only create the orphan packaging branch. Thus, you need to create the development branch with gbs devel start.  
+  The convert action only creates the orphan packaging branch. This means that you need to create the development branch separately with the `gbs devel start` command.  
 
-## Parameters
-
-### Optional Parameters for Subcommands
-
-```
---retain-history Preserve the history of the local changes.
-```
 
-## Using gbs-devel with gbs-build
+## Using gbs devel with gbs build
 
-A recommended procedure that vividly showcases the co-operation of gbs-devel and gbs-build GBS subcommands is shown below, acl package is taken as example here:
+The following example shows a recommended procedure that shows how the `gbs devel` and `gbs build` subcommands work together. The "acl" package is used as an example.
 
-1. Show the branch in joint-packaging model.
+1. Show the branch in the joint-packaging model:
```

Comment:
ordered list

Reply:
As agreed earlier, this numbering will remain, but we will use running numbers in the future.